share/saml-example/create-and-work-with-uni.md

```diff
@@ -0,0 +1,92 @@
+
+First, we will Pick a unique `name` for your uni that begins with `test-` - by default all Unis share a common namespace so here is your chance to get creative.  Update the `userId` attribute of each node to reflect your Vendia Share email address.
+
+Second, update the `authorizerArn` to reflect the ARN of the Amazon Cognito user pool that we [created in our previous step](./auth0-saml-provider-amazon-cup.md).
+
+```bash
+cd uni_configuration
+share uni create --config registration.json
+```
+
+The Uni will take approximately 5 minutes to deploy.  We can check on its status in the Vendia Share web application or with the `share` CLI.
+
+**NOTE:** The name of your Uni will be different.  Adjust as appropriate.
+
+```bash
+share get --uni test-saml-example
+```
+
+Make note of the `httpsUrl` of the **DistributionCenter** when the Uni gets to a `RUNNING` state. We will need it when interacting with our Uni.
+
+# Working with Data in Our Uni
+
+Let's go ahead and set shell variables we'll be using to read and write data.
+
+```bash
+% export GRAPHQL_URL="your-graphql-url"
+
+% export AUTHORIZATION_HEADER="id_token_from_previous_step"
+```
+
+If the `id_token` has expired you will receive the error `{"message":"The incoming token has expired"}` when running GraphQL operations. 
+
+If the `Authorization` header is not present you will receive the error `{"message":"Unauthorized"}` when running GraphQL operations.
+
+## Getting Information About InventoryItems
+
+We have the ability to determine the properties of `InventoryItems` using GraphQL introspection.
+
+```bash
+% curl ${GRAPHQL_URL} \
+-H 'Content-Type: application/json' \
+-H "Authorization: ${AUTHORIZATION}" \
+--data-binary '{"query":"query InventoryItem {\n  __type(name: \"InventoryItem\") {\n    name\n    fields {\n      name\n      type {\n        name\n        kind\n      }\n    }\n  }\n}","variables":{}}' \
+--compressed 
+```
+
+## Reading Data from Our DistributionCenter Node
+
+Our Uni will not have any `InventoryItems` in it since we did not populate it with `initState` data. However, we can still issue a valid GraphQL query to confirm authentication works as expected. We will need to make a request of our **DistributionCenter** `httpsUrl` GraphQL endpoint. This can be done using a GUI like the [Altair GraphQL Client](https://altair.sirmuel.design/) or even a command line utility like [curl](https://man7.org/linux/man-pages/man1/curl.1.html). The `Authorization` header must be present regardless of the utility used to make the GraphQL query.
+
+```bash
+% curl ${GRAPHQL_URL} \
+-H 'Content-Type: application/json' \
+-H "Authorization: ${AUTHORIZATION}" \
+--data-binary '{"query": "query listAllProducts {\n  listInventoryItems {\n    InventoryItems {\n      _id\n      itemName\n      itemNumber\n      quantity\n    }\n  }\n}","variables":{}}' \
```

Comment:
I'd double check this (if you haven't already) against the latest/greatest version of Share.  Not sure how the "Self_" namespace will impact the GQL here, if at all.

Reply:
Good call - the queries worked before the push but they needed to be updated.